File: packages/smartest-backend/src/main/java/com/smartest/backend/constants/QuizPublicationLimits.java
package com.smartest.backend.constants;

/**
 * Publication web du quiz : liste d'emails autorisés stockée côté serveur.
 * Plafond pour l'import / l'assignation des étudiants autorisés à un quiz publié.
 */
public final class QuizPublicationLimits {

    private QuizPublicationLimits() {
    }

    public static final int MAX_AUTHORIZED_STUDENT_EMAILS = 2000;
}

File: packages/smartest-desktop/Constants/QuizPublicationLimits.cs
namespace smartest_desktop.Constants
{
    /// <summary>
    /// Règles produit : publication web du quiz avec liste d'étudiants autorisés (emails).
    /// La liste est persistée côté serveur au moment de « Publier sur le web » (hors périmètre module).
    /// </summary>
    public static class QuizPublicationLimits
    {
        public const int MaxAuthorizedStudentEmails = 2000;
    }
}
